spec/sdr_client/deposit/process_spec.rb

```diff
@@ -95,81 +167,7 @@
-                type: Cocina::Models::ObjectType.book,
-                label: 'This is my object',
-                version: 1,
-                access: {
-                  view: 'world',
-                  download: 'none',
-                  controlledDigitalLending: false
-                },
-                administrative: {
-                  hasAdminPolicy: 'druid:bc123df4567',
-                  releaseTags: []
-                },
-                identification: {
-                  catalogLinks: [],
-                  sourceId: 'googlebooks:12345'
-                },
-                structural: {
-                  contains: [{
-                    type: Cocina::Models::FileSetType.file,
-                    label: 'Page 1',
-                    version: 1,
-                    structural: {
-                      contains: [{
-                        type: Cocina::Models::ObjectType.file,
-                        label: 'file1.txt',
-                        filename: 'file1.txt',
-                        version: 1,
-                        externalIdentifier: 'BaHBLZz09Iiw',
-                        hasMessageDigests: [],
-                        access: {
-                          view: 'world',
-                          download: 'none',
-                          controlledDigitalLending: false
-                        },
-                        administrative: {
-                          publish: true,
-                          sdrPreserve: true,
-                          shelve: true
-                        }
-                      }]
-                    }
-                  }, {
-                    type: Cocina::Models::FileSetType.file,
-                    label: 'Page 2',
-                    version: 1,
-                    structural: {
-                      contains: [{
-                        type: Cocina::Models::ObjectType.file,
-                        label: 'file2.txt',
-                        filename: 'file2.txt',
-                        version: 1,
-                        externalIdentifier: 'dz09IiwiZXhwIjpudWxsLC',
-                        hasMessageDigests: [],
-                        access: {
-                          view: 'world',
-                          download: 'none',
-                          controlledDigitalLending: false
-                        },
-                        administrative: {
-                          publish: true,
-                          sdrPreserve: true,
-                          shelve: true
-                        }
-                      }]
-                    }
-                  }],
-                  hasMemberOrders: [],
-                  isMemberOf: ['druid:gh123df4567']
-                }
-              }.to_json,
-              headers: { 'Content-Type' => 'application/json' }
-            )
+            .with(body: request_dro.to_h.except(:description).to_json, headers: { 'Content-Type' => 'application/json' })
```

Comment:
i'll admit i traced through the code for a bit, and didn't really understand why i needed to omit `description` here and elsewhere in this spec file.  is this ok, or am i papering over a mistake?

Reply:
Looking at the code, it appears that sdr-client does not provide `description` on a request, so I think you're OK.

Reply:
thanks!